### DISCUSSION TITLE
How to Control Shift S

### DISCUSSION THREAD

**Discussion by alasta4204** [AntiMicroX/antimicrox#210] [Q&A]
Good afternoon and thank you everyone for you work on this app .I have installed it today easily ubuntu 20.04 from the flatpak

What I am trying to do is get the Firestorm Second Life viewer to take a silent snapshot ( Control Shift S ) using a X-box 360 controller . I have successfully used the join function to do the Control and Shift keys , but cant get all three keys assigned to one button .

Any help greatly appreciated 

Alastair

**Comment by maxigaz** [comment]
I could map `Ctrl-Shift-v` on an Xbox 360 Controller (This is what I could test easily, but I suppose the same goes for `Ctrl-Shift-c`.) Here’s how I did it.

In the following window, click on `CTRL (L)` and then “Advanced”.

![antimicro-new-assignment](https://user-images.githubusercontent.com/10419911/106963423-da1f8a00-6740-11eb-96bc-255b872b0911.png)

Then next to `Ctrl`, add `Shift` and then `v` by clicking on the button with no label, without changing any options, like so.

![antimicro-adv](https://user-images.githubusercontent.com/10419911/106963418-d986f380-6740-11eb-9614-726523f97bd5.png)

**Comment by alasta4204** [comment]
Many thanks for taking the time to reply !  I tried that , with the Firestorm SL viewer and no joy . I had to use the JOIN button to connect Cntrl And Shift together , couldnt get one button to do all three . I suspect the culprit is the viewer itself , the joystick handling is hopeless and there is no provision for key mapping .As it stands I can activate the feature I want from two button presses on the controller , so this is better than what I had :)
